include/ui/input-text-model.hpp

```diff
@@ -25,7 +25,9 @@
 
 #include "core/memory-access.hpp"
 
-using length_t = unsigned ;
+namespace InputText{
+    using length_t = unsigned ;
```

Comment:
I could not use InputTextModel::length_t in ui.cpp for updating qml. This way it works